sdk/openai/openai/samples/v1-beta/javascript/bringYourOwnData.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why do we see these files here in JavaScript only and not reflected in TypeScript?